uwp/RichTextBox/Hyperlink.md

```diff
@@ -61,6 +61,128 @@ The following code example illustrates how to insert hyperlink field into SfRich
+    </RichTextBoxAdv:SpanAdv>
+    <RichTextBoxAdv:FieldEndAdv></RichTextBoxAdv:FieldEndAdv>
+</RichTextBoxAdv:ParagraphAdv>
+
+{% endhighlight %}
+{% highlight c# %}
+ParagraphAdv paragraphAdv = new ParagraphAdv();
+// Appends the field start.
+paragraphAdv.Inlines.Add(new FieldBeginAdv());
+// Appends the field code.
+SpanAdv fieldCode = new SpanAdv();
+string url = "www.syncfusion.com";
+string screenTip = "SfRichTextBox";
+fieldCode.Text = " HYPERLINK \"" + url + "\" \\o \"" + screenTip + "\" ";
+paragraphAdv.Inlines.Add(fieldCode);
+// Appends the field separator
+paragraphAdv.Inlines.Add(new FieldSeparatorAdv());
+// Appends the field result.
+SpanAdv fieldResult = new SpanAdv();
+fieldResult.Text = "SfRichTextBoxAdv";
+fieldResult.CharacterFormat.Underline = Underline.Single;
+fieldResult.CharacterFormat.FontColor = Color.FromArgb(0xff, 0x05, 0x63, 0xc1);
+paragraphAdv.Inlines.Add(fieldResult);
+// Appends the field end.
+paragraphAdv.Inlines.Add(new FieldEndAdv());
+richTextBoxAdv.Document.Sections[0].Blocks.Add(paragraphAdv);
+
+{% endhighlight %}
+{% highlight VB %}
+' Appends the field start.
+paragraphAdv.Inlines.Add(New FieldBeginAdv())
+' Appends the field code.
+Dim fieldCode As New SpanAdv()
+Dim url As String = "www.syncfusion.com"
+Dim screenTip As String = "SfRichTextBox"
+fieldCode.Text = (Convert.ToString(" HYPERLINK """) & url) + """ ""o """ + (screenTip) + """ "
+paragraphAdv.Inlines.Add(fieldCode)
+' Appends the field separator
+paragraphAdv.Inlines.Add(New FieldSeparatorAdv())
+' Appends the field result.
+Dim fieldResult As New SpanAdv()
+fieldResult.Text = "SfRichTextBoxAdv"
+fieldResult.CharacterFormat.Underline = Underline.[Single]
+fieldResult.CharacterFormat.FontColor = Color.FromArgb(&Hff, &H5, &H63, &Hc1)
+paragraphAdv.Inlines.Add(fieldResult)
+' Appends the field end.
+paragraphAdv.Inlines.Add(New FieldEndAdv())
+
+{% endhighlight %}
+{% endtabs %}
+
+The following code example illustrates how to insert hyperlink field with ScreenTip into RichTextBox Document through UI command.
+{% tabs %}
+{% highlight C# %}
+richTextBoxAdv.InsertHyperlinkCommand.Execute(new string[3] { "www.syncfusion.com", "SfRichTextBoxAdv", "SfRichTextBox" });
+
+{% endhighlight %}
+{% endtabs %}
+
+The following section illustrates how to insert hyperlink field with ScreenTip in RichTextBox Document through built-in hyperlink dialog UI command like Microsoft Word application.
+1. Open insert hyperlink dialog.
+
+{% capture codesnippet1 %}
+{% tabs %}
+{% highlight xaml %}
+<!-- Binds button to the ShowHyperlinkDialogCommand -->
+<Button Content="Hyperlink" Command="{Binding ElementName=richTextBoxAdv, Path=ShowHyperlinkDialogCommand}"/>
+
+{% endhighlight %}
+{% endtabs %}
+{% endcapture %}
+{{ codesnippet1 | OrderList_Indent_Level_1 }}
+
+2. Enter the display text, URL link and ScreenTip text.
+3. Click on OK to close the dialog box.
+
+![Adding Hyperlink to WPF RichTextBox](Hyperlink_images/wpf-richtextbox-insert-hyperlink.PNG)
```

Comment:
Rename the image with correct the platform.

Reply:
Renamed the image with correct platform